src/components/WwwFrame/PromotionalBanner/Banners/DepositIncentiveBanner.vue

```diff
@@ -47,10 +54,14 @@ export default {
 	apollo: {
 		query: amountToLendQuery,
 		preFetch: true,
+		variables: {
+			campaignId: '04786358-043c-4c09-af50-2d5e79ceeacd'
```

Comment:
Does this campaign ID need to be static?

Reply:
@dyersituations I believe so. I got it from the admin dashboard

Reply:
Is this banner only ever going to be used for one campaign? Or is this campaign re-used several times? We don't generally hard-code campaign IDs in our FE code.

Reply:
@dyersituations This banner file is for the deposit incentive.  I'm not very experienced with the FE, what would be the alternative to hard coding the campaign Id?

Reply:
The general question I have is: is the deposit incentive a single campaign that is being ran only once? And is this banner only going to be used for this single campaign?

Reply:
@dyersituations Yes, I believe its a one off campaign. But if its re-ran in the future, I would assume they would use the same campaign and the id wouldnt change. cc @emuvente

Reply:
I think we'll run a new deposit incentive campaign, so it would be better if the campaign id was fetched from settings manager. That would require a more complicated prefetch though, so we can assign this ticket to a frontend engineer instead if you would prefer @mlibazisi?

Reply:
@emuvente Sure, that works!

Reply:
@emuvente Maybe we can merge this ticket, then have that as a follow up ticket?

Reply:
@mlibazisi Sure, we can do that